src/js/protocols/stm32usbdfu.js

```diff
@@ -149,17 +148,19 @@ STM32DFU_protocol.prototype.claimInterface = function (interfaceNumber) {
         // Don't perform the error check on MacOS at this time as there seems to be a bug
         // where it always reports the Chrome error "Error claiming interface." even though
         // the interface is in fact successfully claimed.
-        if (checkChromeRuntimeError() && (GUI.operating_system !== "MacOS")) {
-            console.log('Failed to claim USB device!');
+        if (checkChromeRuntimeError()) {
```

Comment:
This check doesn't seem to be needed.

After removing the check....
- When the check fails, ie when the USB interface cannot be 'claimed', the flash will fail.
- When the check is succesfully passed, it flashes properly.

This seems to be the correct behaviour